CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * **BUGFIX** (by @imaNNeo) Show `0` instead of `-0` in some edge-cases in the default titles 
 * **FEATURE** (by @tamasapps): Add `tooltipHorizontalAlignment` and `tooltipHorizontalOffset` property in [LineTouchTooltipData], [BarTouchTooltipData], [ScatterTouchTooltipData].
 * **FEATURE** (by @dhiyaaulauliyaa) Add ability to force SideTitle to be placed inside its corresponding axis bounding box, #603.
+* **BUGFIX** (by @JoshMart) Fix horizontal lines not painting when at chart min or max, #1255.
```

Comment:
0.61.0 was the previous release.
Please read the [contribution guideline](https://github.com/imaNNeo/fl_chart/blob/master/CONTRIBUTING.md) carefully.

You need to create a new release on the top called `newRelease`. Like the below code:

```
## newRelease
* **BUGFIX** (by @JoshMart) Fix horizontal lines not painting when at chart min or max, #1255.
```

Reply:
Good catch, thanks. Got this updated.